src/styles/components/_Footer.scss

```diff
@@ -38,8 +38,16 @@
     }
   }
 
-  &::after {
-    content: '·';
+  &:not(:last-child)::after {
+    --marker-size: 2px;
```

Comment:
This works for me! I'm wondering about using this new pattern for variable naming and use - We've not be using `--var` and `height: var(--marker-size)` and I think we could definitely adopt this approach. I'd want to make sure we're using the same style across the board for consistency and readability.

Reply:
I went ahead and refactored it to use a Sass variable. Wasn't sure where I was supposed to put it, so I added it to the top of the file. Assuming that's ok?